hack/verify.sh

```diff
@@ -1,38 +1,44 @@
 #!/bin/bash
 
-export CRTDIR=$(pwd)
-export TMPDIR=/tmp/testgendocs
-mkdir $TMPDIR
+CRT_DIR=$(pwd)
+VERIFY_TEMP=$(mktemp -d 2>/dev/null || mktemp -d -t k8s-community.XXXXXX)
+WORKING_DIR=${VERIFY_TEMP}/src/testgendocs
+GOPATH=${VERIFY_TEMP}
+mkdir -p ${WORKING_DIR}
 
-cp -r sig* Makefile generator $TMPDIR
+function cleanup {
+  rm -rf "${VERIFY_TEMP}"
+}
+trap cleanup EXIT
 
-cd $TMPDIR
+cp -r sig* Makefile generator ${WORKING_DIR}
```

Comment:
We also have wg-* and we should take this into account here too.  Looks like that perhaps got lots when the wg* stuff was added.

Reply:
Good catch. Fixing.